src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3371,7 +3371,7 @@ mobile_apis::Result::eType ApplicationManagerImpl::CheckPolicyPermissions(
       ", window_id: " + std::to_string(window_id) +
       ", HMI status: " + EnumToString(app->hmi_level(window_id));
 #endif  // ENABLE_LOG
-  if (result.hmi_level_permitted != policy::kRpcAllowed) {
+  /*if (result.hmi_level_permitted != policy::kRpcAllowed) {
```

Comment:
@VadymLuchko what is this?

Reply:
oh, it result of the rush
removed in 7f32022e5